eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,107 @@
+$Language = "python"
+$PackageRepository = "PyPI"
```

Comment:
Unless we feel PackageRepository is interesting I'd suggest we not include it.

Reply:
Its still used in various places. I think we can leave it for now. I can remove it when working on the other scripts if it proves unnecessary.